api/src/pcapi/core/offerers/api.py

```diff
@@ -619,7 +619,10 @@ def set_offerer_attachment_pending(
 
 
 def reject_offerer_attachment(
-    user_offerer: offerers_models.UserOfferer, author_user: users_models.User, comment: str | None = None
+    user_offerer: offerers_models.UserOfferer,
+    author_user: users_models.User,
+    comment: str | None = None,
+    with_email: bool | None = True,
```

Comment:
Pourquoi `with_email` pourrait valoir `None` ? Soit on passe un booléen, soit on ne passe pas de valeur (et la valeur par défaut est un booléen), non ?

```suggestion
    with_email: bool = True,
```

Reply:
Oui je me suis un peu perdu sur celui-ci